samples/maps/demo/geoheatmap-earth-statistics/demo.js

```diff
@@ -0,0 +1,224 @@
+                        datasetSelect = document.getElementById('dataset');
+
+                    datasets.forEach(el => {
+                        const option = document.createElement('option');
+                        option.value = el.type;
+                        option.innerHTML = el.type;
+                        datasetSelect.appendChild(option);
+                    });
+
+                    // Show the Font Awesome spinner
+                    chart.showLoading('<i class="icon-spinner icon-spin icon-3x"></i><br/><i>Loading data...</i>');
+
+                    async function getDataset(type) {
+                        const dataset = datasets.find(el => el.type === type);
+
+                        if (typeof dataset.data === 'undefined') {
+                            const apiData = await fetch(dataset.url)
+                                .then(response => response.json());
+                            dataset.data = apiData;
+                        }
+
+                        // Hide loading
+                        chart.hideLoading();
+                        chart.title.update({
+                            text: dataset.title
+                        }, false);
+                        chart.colorAxis[0].update(dataset.colorAxis, false);
+                        geoheatmap.setData(dataset.data);
+                    }
+
+                    datasetSelect.addEventListener('change', function () {
+                        // Show the Font Awesome spinner
+                        chart.showLoading('<i class="icon-spinner icon-spin icon-3x"></i><br/><i>Loading data...</i>');
+                        getDataset(datasetSelect.value);
+                    });
+
+                    getDataset(datasetSelect.value);
+                }
+            }
+        },
+
+        title: {
+            text: 'Land Surface Temperature (day) in August 2022',
+            style: {
+                color: '#fff'
+            }
+        },
+
+        subtitle: {
+            text: 'Data source: <a style="color: #ddd" href="https://neo.gsfc.nasa.gov/">NEO Nasa Earth Observations</a>',
+            style: {
+                color: '#fff'
+            }
+        },
+
+        mapNavigation: {
+            enabled: true,
+            buttonOptions: {
+                verticalAlign: 'bottom'
+            }
+        },
+
+        mapView: {
+            center: [0, 0],
+            zoom: 1.5
+        },
+
+        legend: {
+            symbolWidth: 350
+        },
+
+        loading: {
+            labelStyle: {
+                color: 'white'
+            },
+            style: {
+                backgroundColor: 'rgba(0,0,0,0)'
+            }
+        },
+
+        colorAxis: {},
+
+        tooltip: {
+            headerFormat: '<span style="font-size: 11px">Lon: {point.point.lon}° Lat: {point.point.lat}°</span><br/>',
+            pointFormatter() {
+                return `Value: ${this.options.value.toFixed(2)}`;
+            }
```

Comment:
Avoid unnecessary use of formatter callbacks. Use this instead:

`pointFormat: 'Value: {point.value:.2f}'`